src/Microsoft.DotNet.Cli.Utils/Product.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Reflection;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class Product
+    {
+        public static readonly string LongName = ".NET Command Line Tools";
+        public static readonly string Version = GetProductVersion();
+
+        private static string GetProductVersion()
+        {
+            var attr = typeof(Product).GetTypeInfo().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>();
```

Comment:
yes, @LakshanF changed --version to call into Product.Version.  This code was just moved from the HelpCommand class.

See
https://github.com/dotnet/cli/pull/2066/files#diff-b792a66d46e9b54698fc1524afcfd69fR164
and
https://github.com/dotnet/cli/pull/2066/files#diff-b483d53e53136ba110ff13736c88674eL35